app/services/food_service.rb

```diff
@@ -0,0 +1,23 @@
+class FoodService
+  def results(q)
+    response = conn.get("/fdc/v1/foods/search?") do |faraday|
+      faraday.params['query'] = q
```

Comment:
Love that you're setting your query params here in the block. Readable and maintainable.